src/types.ts

```diff
@@ -884,14 +890,15 @@ export type SupportedExchanges = "alby" | "coindesk" | "yadio";
 
 export interface Invoice {
```

Comment:
I think this type should be renamed for consistency or removed if possible. Couldn't the ConnectorTransaction already decode the custom_records? 

Transaction: Invoice + Payment
ConnectorTransaction: Invoice + Payment on connector level

Reply:
we still need Invoice type for getInvoice method as it returns invoice itself and not connectorTRansaction